deluge/core/torrent.py

```diff
@@ -685,7 +687,7 @@ def force_error_state(self, message, restart_to_resume=True):
             restart_to_resume (bool, optional): Prevent resuming clearing the error, only restarting
                 session can resume.
         """
-        status = self.handle.status()
+        status = self.update_status()
```

Comment:
Naming of `update_status` seems misleading, `get_status` or `fetch_status` perhaps? Maybe with a `cached=False` arg?

Reply:
Yeah, I did `get_status` at first, until I realized there already is a `get_status` which is a combined options and status dict. Maybe rename `torrent.status` cache to `torrent.lt_status`, and have `torrent.get_lt_status`?

Reply:
So I did the rename to `torrent.lt_status`, with the refresh option now being `torrent.get_lt_status()`. I think this helps clarify the difference between the libtorrent object and the deluge status dict, which is grabbed by `torrent.get_status()`. Happy to do it a different way though if anyone has better ideas.

Reply:
Yeah I did want to think about this some more as I am not sure that we should be exposing torrent handle status outside of the Torrent class

Reply:
Yeah, I was thinking that as well. It's only used in a few spots in torrentmanager, which could probably be updated.

Reply:
The one spot in torrentmanager where it seems sorta important is here https://github.com/deluge-torrent/deluge/pull/369/files#diff-3416e08006d0ef78d79619b83bf6d42c888dc1e5b20ea03801a71e357d5484c9R1615

Which is where it handles the bulk update alert and updates the status cache of all the torrents. I think this was one of the things that improved performance on large numbers of torrents.

Reply:
I am not sure about wholesale renaming here, perhaps we keep `status` attribute for now and keep new `get_lt_status`?

Reply:
That seems good to me too. I think perhaps `status` could/should be renamed and become private in the future, (maybe `self._lt_status`,) but I don't think that needs to be tied up in this bugfix. I'll fix the PR up when I get a chance.

Reply:
Reverted the `.status` rename, but kept the `.get_lt_status()` now.